front/src/applications/stdcm/utils/createMargin.ts

```diff
@@ -0,0 +1,17 @@
+import type { MarginValue } from 'common/api/osrdEditoastApi';
+import type { StandardAllowance } from 'reducers/osrdconf/types';
+
+export default function createMargin(margin: StandardAllowance | undefined): MarginValue {
+  if (!margin || !margin.value) {
+    return 'None';
+  }
+  switch (margin.type) {
+    case 'time_per_distance':
+      return `${margin.value}min/km` as unknown as MarginValue;
+
+    case 'percentage':
+      return `${margin.value}%` as unknown as MarginValue;
+    default:
+      throw new Error(`Unsupported value_type: ${margin.type}`);
+  }
+}
```

Comment:
It seems you didn't rebase with last back changes for the stdcmV2 endpoint. `MarginValue` no longer exists and `margin` is now a string (as you almost properly did actually)
```suggestion
export default function createMargin(margin: StandardAllowance | undefined): string {
  if (!margin || !margin.value) {
    return 'None';
  }
  switch (margin.type) {
    case 'time_per_distance':
      return `${margin.value}min/100km`;

    case 'percentage':
      return `${margin.value}%`;
    default:
      throw new Error(`Unsupported value_type: ${margin.type}`);
  }
}
```

Reply:
Done ! I didn't know it was already fixed, great